spec/classroom_spec.rb

```diff
@@ -0,0 +1,8 @@
+require_relative '../classroom'
+
+RSpec.describe Classroom do
+  it 'creates a new classroom' do
+    classroom = Classroom.new('A101')
+    expect(classroom.label).to eq('A101')
+  end
+end
```

Comment:
- [ ] Please make sure to add a test for the `add_student` method found inside the `Classroom` class to ensure your tests cover all the methods as per the project requirements.